src/sidebar/components/version-info.js

```diff
@@ -38,13 +38,11 @@ function VersionInfo({ flash, versionData }) {
         <dd className="version-info__value">{versionData.timestamp}</dd>
       </dl>
       <div className="version-info__actions">
-        <button
-          className="version-info__copy-btn action-button"
```

Comment:
It looks like the `version-info__copy-btn` style was never defined.

Reply:
So removing it should be fine, then, eh? :)